rush.json

```diff
@@ -282,25 +282,25 @@
       "projectFolder": "misk-web/web/packages/@misk/tslint",
       "versionPolicyName": "MiskWeb"
     },
-    {
-      "packageName": "misk-web-examples-data",
-      "projectFolder": "examples/data",
-      "versionPolicyName": "MiskWebDemo"
-    },
-    {
-      "packageName": "misk-web-tab-dashboard",
-      "projectFolder": "examples/services/dashboard",
-      "versionPolicyName": "MiskWebDemo"
-    },
+    // {
+    //   "packageName": "misk-web-examples-data",
+    //   "projectFolder": "examples/data",
+    //   "shouldPublish": false
+    // },
+    // {
+    //   "packageName": "misk-web-tab-dashboard",
+    //   "projectFolder": "examples/services/dashboard",
+    //   "shouldPublish": false
+    // },
     {
       "packageName": "misk-web-tab-palette",
       "projectFolder": "examples/tabs/palette",
-      "versionPolicyName": "MiskWebDemo"
+      "shouldPublish": false
     },
     {
       "packageName": "misk-web-tab-palette-exemplar",
       "projectFolder": "examples/tabs/palette-exemplar",
-      "versionPolicyName": "MiskWebDemo"
+      "shouldPublish": false
     }
     // {

```

Comment:
what's going on here?

Reply:
Part of the process of moving the `examples/*` projects into the new main Rush project. Each package registration can have either `shouldPublish` or `versionPolicyName`. The `shouldPublish: false` ensures that example projects aren't published to NPM (since they don't need to be).